Macros/ACS_summary_geo.sas

```diff
@@ -520,12 +483,31 @@
        mNumRentCstBurden_20_34K_&_years. mNumRentCstBurden_35_49K_&_years. 
        mNumRentCstBurden_50_74K_&_years. mNumRentCstBurden_75_99K_&_years. 
        mNumRentCstBurden_GT100K_&_years. 
-
-       mNumOwnCstBurden_LT10K_&_years. mNumOwnCstBurden_10_19K_&_years. 

```

Comment:
just double checking this section is in the new code?

Reply:
@lhendey I'm not sure I understand what you mean by 'in the new code' ?

Reply:
@rpitingolo  i i think i was just double check in your current version of summary_geo that those vars are there- it was unclear to me in the side by side comparison if they just got moved around or not.

Reply:
@lhendey Oh OK. I see what you are saying. I'll check.

Reply:
@lhendey they are still there. In the %let moe2013plus = list. That's where you want them, right?

Reply:
@rpitingolo fine.